back-end/src/main/java/com/skynet/skymed/model/Pessoa.java

```diff
@@ -53,31 +55,50 @@ public String getNome() {
 		return nome;
 	}
 
-	public void setNome(String nome) {
+	public void setNome(String nome) throws Exception {
+		if (nome == null) {
+			throw new Exception("Nome inválido.");
+		}
+		if (nome.isEmpty() || nome.length() < 5) {
```

Comment:
```suggestion
		if (nome.isEmpty()) {
```
Tamanho máximo do campo nome é 5 caracteres?

Reply:
Te pergunto o mesmo, vc colocou na classe medico